pages/common/date.md

```diff
@@ -21,3 +21,9 @@
 - Convert a specific date to the Unix timestamp format:
 
 `date -d "{{2018-09-01 00:00}}" +%s --utc`
+
+- Display RFC-3339, great for logging and timestamps YYYY-MM-DD hh:mm:ss TZ
+
+`date ---rfc-3339=s`
```

Comment:
This example doesn't work for me on Ubuntu 19.10. What system are you using?
```
$ date ---rfc-3339=s
date: unrecognised option '---rfc-3339=s'
Try 'date --help' for more information.
```

Reply:
It should be with `--`, not `---`. I edited the file to correct those pesky errors.

Reply:
Ah, I see. Thanks @zdroid :-)